#!/usr/bin/env python
PACKAGE = "vision_module"

from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()
 
gen.add("DISPLAY", bool_t, 0, "DISPLAY RESULT", False)
gen.add("SAVE_IMAGE", bool_t, 0, "SAVE IMAGE", False)

exit(gen.generate(PACKAGE, "face_detection", "face_detection"))
